src/modules/site-v2/templates/releases/download_tab_isotype_v1.html

```diff
@@ -54,7 +54,7 @@ <h4>VCF</h4>
                 Strain Data including isolation location, isotype information, and more is available for download.
                 <br />
                 <br />
-                <a href="{{ url_for('request_strains.strains_data_csv', release_name = release_version, species_name = species.name, file_ext='csv') }}" type="button" class="btn btn-default btn-block btn-success">Download Strain Data</a>
+                <a href="{{ url_for('request_strains.strains_data_csv', release_name = release_version, species_name = species.get_url_name(), file_ext='csv') }}" type="button" class="btn btn-default btn-block btn-success">Download Strain Data</a>

```

Comment:
Could we find a more descriptive or intuitive name for `#get_url_name()` (eg: referring to prefix/suffix/slug/etc)?

Reply:
Updated in [6617454](https://github.com/AndersenLab/CAENDR/pull/373/commits/6617454bc096bc8048042cab2a966f997abc6fe6) to `get_slug()`